libvast/src/system/spawn_exporter.cpp

```diff
@@ -28,18 +31,85 @@
 
 namespace vast::system {
 
+namespace {
+
+caf::expected<std::pair<expression, std::optional<pipeline>>>
+parse_arguments(const std::vector<std::string>& args) {
+  static const auto match_everything = expression{predicate{
+    meta_extractor{meta_extractor::kind::type},
+    relational_operator::not_equal,
+    data{"this expression matches everything"},
+  }};
+  if (args.empty()) {
+    return std::pair{
+      match_everything,
+      std::optional<pipeline>{},
+    };
+  }
+  const auto repr = detail::join(args.begin(), args.end(), " ");
+  using parsers::space, parsers::expr, parsers::eoi;
```

Comment:
Should we use blank over space here?

Reply:
Replaced in https://github.com/tenzir/vast/pull/2877/commits/92595e7450a00df6fb03e8c9458d6aec21a8e58c